pages/data-modeling/getting-setup.mdx

```diff
@@ -0,0 +1,40 @@
+# Getting Set Up for Data Modeling
+
+You can define your data models in individual files using **YAML** or **JavaScript**. Generally, YAML is recommended for its simplicity and readability. (See [Cube’s documentation](https://cube.dev/docs) if you prefer JavaScript-based models.)
```

Comment:
@hjkmarshall hey sir - i find it's better to link to `https://cube.dev/docs/product/data-modeling/overview` than `cube.dev/docs` as their /docs page talks about all their features, many of whom are not relevant to Embeddable (and thus, in my experience, causes confusion), whereas the /data-modeling/overview goes directly to the part that's relevant to embeddable